docs/docs-components/AppLayout.js

```diff
@@ -15,16 +16,26 @@ const CONTENT_MAX_WIDTH_PX = 1546;
 type Props = {|
   children?: Node,
   colorScheme?: 'light' | 'dark',
-  isHomePage?: boolean,
 |};
 
-export default function AppLayout({ children, colorScheme, isHomePage }: Props): Node {
+export default function AppLayout({ children, colorScheme }: Props): Node {
   const { isMobile } = useDocsDeviceType();
   const { isSidebarOpen, setIsSidebarOpen } = useNavigationContext();
+  const router = useRouter();
+
+  const [isHomePage, setIsHomePage] = useState(router?.route === '/home');
```

Comment:
Does this need to be a state variable? It seems like it could be a simple check (`router?.route === '/home'`) in the main body of the function component. I'm also confused why we're referencing `router.route` here and `window.location.pathname` below